modules/ROOT/pages/to-customize-portal.adoc

```diff
@@ -72,6 +72,18 @@ To delete a custom page, click *Delete Page*. If this button is not visible resi
 
 To change the order of the custom page tabs, drag the entries in the *Custom pages* list into the desired order.
 
+== Custom Login
```

Comment:
I think this title is still confusing, since it gives the impression that the Login is customizable in itself, what are your thoughts?

Reply:
Good point. Updated to "Login Page".